tools/wheelmaker.py

```diff
@@ -252,6 +261,11 @@ def main():
         help='A file that has all the input files defined as a list to avoid the long command'
     )
 
+    packages_group = parser.add_argument_group("Packages")
+    packages_group.add_argument(
+        '--package', type=str, action='append',
+        help="List of packages the distribution provides. Can be supplied multiple times.")
```

Comment:
include the defaulting logic in this doc: "if not specified, we assume there is a single top-level package named after the name  attribute"